samples/nrf9160/coap_client/prj.conf

```diff
@@ -0,0 +1,29 @@
+# General config
+CONFIG_TEST_RANDOM_GENERATOR=y
+CONFIG_HEAP_MEM_POOL_SIZE=16384
+
+# Networking
+CONFIG_NETWORKING=y
+CONFIG_NET_SOCKETS_OFFLOAD=y
+CONFIG_NET_SOCKETS=y
+CONFIG_NET_SOCKETS_POSIX_NAMES=y
+
+# LTE link control
+CONFIG_LTE_LINK_CONTROL=y
+CONFIG_LTE_AUTO_INIT_AND_CONNECT=n
+
+# BSD library
+CONFIG_BSD_LIBRARY=y
+CONFIG_BSD_LIBRARY_TRACE_ENABLED=n
+
+# AT Host
+CONFIG_UART_INTERRUPT_DRIVEN=y
+CONFIG_AT_HOST_LIBRARY=y
+
+# CoAP
+CONFIG_NRF_COAP_LIB=y
+CONFIG_NRF_COAP_PORT_COUNT=1
+
+# Main thread
+CONFIG_MAIN_THREAD_PRIORITY=7
```

Comment:
This is a typical setting carried over from sample to sample, but not sure where it originates from or why it was introduced in the first place. It works fine with the default priority as well.